src/main.py

```diff
@@ -21,6 +23,19 @@
 import persistent_state
 import song_utils
 
+
+def setup_logging(log_level):
+    logger = logging.getLogger("nextcord")
+    logger.setLevel(log_level)
+    handler = logging.StreamHandler(stream=sys.stderr)
```

Comment:
Streaming all of nextcord's logs to stderr is an interesting choice. Is this to separate the logs from busty's own?

stderr should still be visible in `journalctl` and the command line, so this works for now.

Reply:
Yes this PR was intended to be more of a "very quick, for now" type deal just so if it crashes we're ready. Then we accidentally let it sit for two weeks lol

I'll send a follow-up PR with more thought-out organization